src/sst/core/impl/partitioners/singlepart.h

```diff
@@ -45,13 +45,12 @@ class SSTSinglePartition : public SST::Partition::SSTPartitioner {
        Performs a partition of an SST simulation configuration
        \param graph The simulation configuration to partition
     */
-    void performPartition(PartitionGraph* graph) override;
-
-    void performPartition(ConfigGraph* graph) override {
+    void performPartition(PartitionGraph* graph) override {
         SST::Partition::SSTPartitioner::performPartition(graph);
```

Comment:
Is this code ever intended to be called? My understanding is the because `requiresConfigGraph` returns true this always routes to config graph function, not the partition graph function. This wouldn't actually produce a valid partition, would it? Maybe we don't need to fix for this PR, but would it be more appropriate for `performPartition(PartitionGraph* graph)` to crash and exit?

Reply:
Only one of the versions will ever get called.  The off function is calling the parent function, which does just fatal.  Not sure why it just doesn't declare the one that isn't used because it would do the same thing but the parent function would get called directly.  I suspect the other partitioners are doing the same thing.  It functions the way you suggested, even through it's done in a weird way.  We should probably look at cleaning this up across all partitioners in a different PR.